xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -79,16 +79,30 @@
         this.execution = execution;
         this.componentManager = componentManager;
     }
+    
+    /**
+     * Add some content to the mail to be sent. Can be called several times to add different content type to the mail.
+     * This method can be used for scripting languages other than Velocity
```

Comment:
This line should be removed. Nothing prevent Velocity to use this method as long as some API provide any way to get a BodyPart object. It could even be used directly if a BodyPart Converter is implemented.